src/pages/signin/SignInPage.js

```diff
@@ -92,9 +92,13 @@ class SignInPage extends Component {
                 welcomeText = this.props.translate('validateCodeForm.enterAuthenticatorCode');
             } else {
                 const userLogin = Str.removeSMSDomain(lodashGet(this.props, 'credentials.login', ''));
+
+                // replacing spaces with "hard spaces" to prevent breaking the number
+                const userLoginToDisplay = Str.isSMSLogin(userLogin) ? this.props.formatPhoneNumber(userLogin).replace(/ /g, '\u00A0') : userLogin;
```

Comment:
Since avoiding breaking of number is a style concern, I was about to suggest that we use `white-space: nowrap`.

But then it's part of welcome text and it can get a bit messy to implement.

Reply:
Ideally we can find a more DRY way to do this.

Reply:
I wonder if we could add a function for replacing the spaces to Str util in expensify-common lib?